self_hosted/tokenizer.jou

```diff
@@ -22,7 +22,7 @@ def is_keyword(word: byte*) -> bool:
     #   - syntax documentation
     keywords = [
         "import", "def", "declare", "class", "union", "enum", "global",
-        "return", "if", "elif", "else", "while", "for", "break", "continue",
+        "return", "if", "elif", "else", "while", "for", "break", "continue", "pass", 
```

Comment:
See the comment just above this code. You need to add the `pass` keyword to `doc/syntax-spec.md` where there is a list of all keywords

Reply:
Nevermind, it is there already. I didn't see it for some reason...